src/IO/JSON/JSONIOHandlerImpl.cpp

```diff
@@ -1077,6 +1078,14 @@ namespace openPMD
         {
             case Access::CREATE:
             case Access::READ_WRITE:
+            case Access::APPEND:
+                /*
+                 * Always truncate when writing, we alway write entire JSON
+                 * datasets, never partial ones.
```

Comment:
Writing with `storeChunk` can still be _partial_, even in serial writes. Thus the text here is a bit unclear in the argument, I think.

(I am ok with the logic to truncate.)

Reply:
What I mean in here is that this backend always re-writes the entire JSON file and does not just update parts of it. This makes it ok to just truncate the file. So maybe:
```suggestion
                 * Always truncate when writing, we alway write entire JSON
                 * files, never partial ones.
```